contrib/graph/graph2.py

```diff
@@ -126,7 +130,7 @@ def make_graph(db_name, room_id, file_prefix, limit):
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(
         description="Generate a PDU graph for a given room by talking "
-        "to the given homeserver to get the list of PDUs. \n"
+        "to the given a Synapse SQLite file to get the list of PDUs. \n"
```

Comment:
you might want to either remove either "the given" or "a" here otherwise it's grammatically incorrect-my bias would be towards keeping the "a" and dropping "the given"

Reply:
```suggestion
        "to the given Synapse SQLite file to get the list of PDUs. \n"
```